trin-core/src/cli.rs

```diff
@@ -110,6 +110,37 @@ pub struct TrinConfig {
 
     #[structopt(long = "metrics-url", help = "URL for prometheus server")]
     pub metrics_url: Option<String>,
+
+    #[structopt(
+        short = "e",
+        long = "ephemeral",
+        help = "Enable temporary trin data storage that is deleted on exit."
+    )]
+    pub ephemeral: bool,
+}
+
+impl Default for TrinConfig {
+    fn default() -> Self {
+        TrinConfig {
+            web3_transport: "ipc".to_string(),
+            web3_http_address: DEFAULT_WEB3_HTTP_ADDRESS.to_string(),
+            web3_ipc_path: DEFAULT_WEB3_IPC_PATH.to_string(),
+            pool_size: 2,
+            discovery_port: DEFAULT_DISCOVERY_PORT.parse().unwrap(),
+            bootnodes: vec![],
+            external_addr: None,
+            no_stun: false,
+            private_key: None,
+            networks: DEFAULT_SUBNETWORKS
```

Comment:
I don't really follow this suggestion. Are you suggesting just inlining "history" here? I like it better that it at least shares a constant with the CLI argument default defined earlier.

Reply:
I'm suggesting removing the `.split(',')...` logic below since it's no longer needed now that the constant is just `"history"`. iirc the logic was needed for earlier when the constant was `"history,state"`